api/grpcserver/v2beta1/smeshing_identities.go

```diff
@@ -80,10 +82,10 @@ func (s *SmeshingIdentitiesService) States(
 	return &pb.IdentityStatesResponse{Identities: pbIdentities}, nil
 }
 
-func toEventOperations(filter *pb.IdentityStatesRequest) builder.Operations {
-	ops := builder.Operations{}
+func toEventOperations(filter *pb.IdentityStatesRequest) (*builder.Operations, error) {
```

Comment:
NIT: It's OK to return `builder.Operations{}` as zero value in the case of an error, it doesn't have to be `nil`. Since if a function returns an error that is != nil the returned value in general should not be used.